extensions/amp-list/0.1/amp-list.js

```diff
@@ -602,6 +606,14 @@ export class AmpList extends AMP.BaseElement {
             'update': false,
           });
         }
+        toArray(
+          dev().assertElement(this.container_).children
+        ).forEach((child) =>
+          this.owners_./*OK*/ scheduleUnlayout(
+            dev().assertElement(this.container_),
+            child
```

Comment:
You'd want to pass the `container_` here, and we'll find all amp-element under that container.